src/bootstrap-table.js

```diff
@@ -1569,10 +1596,6 @@
           to = o.paginationSuccessivelySize
         }
 
-        if (to > this.totalPages) {
-          to = this.totalPages
-        }
-
```

Comment:
deleting these columns?

Reply:
Yes!
Check https://github.com/wenzhixin/bootstrap-table/blob/develop/src/bootstrap-table.js#L1584-L1586 and https://github.com/wenzhixin/bootstrap-table/blob/develop/src/bootstrap-table.js#L1596-L1598 code duplication.